### PR TITLE
Add player argument to onVehicleExplode

### PR DESCRIPTION
**Pull request by FileEX**
Closes #3641 

Due to the author's inactivity, I am opening a new PR with the necessary corrections.

### CODE REVIEW

File: Server/mods/deathmatch/logic/CGame.cpp [hunk 1]
```diff
@@ -1633,7 +1633,7 @@ void CGame::AddBuiltInEvents()
     m_Events.AddEvent("onVehicleStartExit", "player, seat, jacker", NULL, false);
     m_Events.AddEvent("onVehicleEnter", "player, seat, jacked", NULL, false);
     m_Events.AddEvent("onVehicleExit", "player, seat, jacker", NULL, false);
-    m_Events.AddEvent("onVehicleExplode", "", NULL, false);
+    m_Events.AddEvent("onVehicleExplode", "withExplosion, player", NULL, false);
```
Comment (TracerDS) [inline Server/mods/deathmatch/logic/CGame.cpp]: can you change `NULL` to `nullptr` while you're at it?

**Review by TracerDS — APPROVED** [review]

**Comment by TheNormalnij** [comment]
btw, do we use predefined variable 'client' in serverside events?

**Comment by TheNormalnij** [comment]
And why not disable an explosion if the explosion source is not a vehicle
syncer.

**Review by TracerDS — APPROVED** [review]

**Comment by Dutchman101** [comment]
> And why not disable an explosion if the explosion source is not a vehicle syncer.

Please open a new PR for that

**Pull request merged**